src/conditions/condition-set.ts

```diff
@@ -11,20 +11,25 @@ export class ConditionSet {
   constructor(public readonly conditions: ReadonlyArray<ConditionOrOperator>) {}
 
   public validate() {
+    // Expects [Condition, Operator, Condition, Operator, ...], where the last element is a Condition
+
     if (this.conditions.length % 2 === 0) {
       throw new Error(
-        'conditions must be odd length, ever other element being an operator'
+        'conditions must be odd length, every other element being an operator'
```

Comment:
In JavaScript, changing the text of an error message is not generally considered a breaking change. Nevertheless, I want to point this out as a functional change.